tasks/discover-granules/tests/index.js

```diff
@@ -70,10 +69,10 @@ test('discover granules using FTP', async (t) => {
     password: 'testpass'
   };
 
-  await validateConfig(t, event.config);
+  validateConfig(t, event.config);
```

Comment:
We can remove `await` from the call to `validateConfig` because it does not return a Promise, but rather a `boolean`. If an async function does not return a Promise, there's no need to use `await` when calling it. Regarding `assertDiscoveredGranules`, it's not an async function.

Reply:
I'm pretty sure we need `await` for `async functions`.  An async function always returns a promise.  `assertDiscoveredGranules` is async as well.  We can discuss in slack

Reply:
Yes, you're right. I was letting the hints in VSCode influence me here. I was seeing hints indicating that use of `await` was superfluous, but after I looked again, I found that the `validateConfig` and `validateOutput` functions had incorrect jsdoc comments, which did not specify a `Promise` as a return value, and VSCode uses the jsdoc comments to provide those hints. Once I fixed the jsdoc comments on those functions to indicate that they return Promises, those hints went away. I will correct all of this to put `await`s back, along with the jsdoc comment fixes.